.storybook/blocks/TabContainer/TabContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, useState } from 'react';
+
+import styles from './TabContainer.module.css';
+
+type Tab = {
+  label: string;
+  component: ReactNode;
+};
+
+type TabContainerProps = {
+  tabs: Tab[];
+};
+
+export const TabContainer: React.FC<TabContainerProps> = ({tabs}) => {
+  const [viewedTab, setViewedTab] = useState<number>(0);
+  const isActive = (index: number): boolean => index === viewedTab;
+
+  if (!tabs.length) return null;
+
+  return (
+    <>
+      <div className={styles.tab_header}>
+        {tabs.map(({ label }, index) => <button className={isActive(index) && styles.active} onClick={() => setViewedTab(index)}>{label}</button>)}
```

Comment:
- you forgot add the `key` for each element of the loop, and please add the esLint rule for it.
- let's define names classes in the beginning of the component the same like in the `packages/button/src/Button.tsx`.
- handle callback should define at the beginning of the component. inside the JSX the callback will always create a new function. 
- the naming of the handler should be `handle_name-element_event` === `handleTabClick`

```
const handleTabClick = () => 
```